src/utils/injectTreeNode.js

```diff
@@ -14,40 +16,46 @@ export default (
     style,
   } = {}
 ) =>
-  injectDefaults(({ tree, node, group, path, ...props }) => {
-    node = node || tree.getNode(path)
+  _.flow(
+    inject(({ theme = {} }) => ({
```

Comment:
@daedalus28 I though `contextureTheme` a bit redundant. After all, we don't do `contextureTree`

Reply:
The idea was that `theme` is more likely to have naming collisions than `tree`, but you're right that it's probably better to be consistent 👍 

At some point, we'll switch to native react context and will reevaluate it anyway